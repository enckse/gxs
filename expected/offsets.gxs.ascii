
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
   - -   -                          
. . . . . . . . . . . . . . . . . . 
     - - a b   b                    
. . . . . . . . . . . . . . . . . . 
       -   b a b                    
. . . . . . . . . . . . . . . . . . 
             a                      
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                       | | | | |    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                       | | | | |    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                                    
. . . . . . . . . . . . . . . . . . 
                                    
---
color: a => orange (count: 3)
color: b => pink (count: 4)
WARN: cannot have stitch+line in ASCII pattern [1]
