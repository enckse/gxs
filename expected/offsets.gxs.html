<!doctype html>
<html lang="en">
<head>
<meta charset="UTF-8">
<style>
.container {
  background: white;
  display: inline-block;
  border: 1px dotted black;
}
.grid {
  display: grid;
  grid-template-columns: repeat(16, 10px);
  grid-template-rows: repeat(16, 10px);
  grid-gap: 1px;
}
.cell {
  justify-content: center;
  align-items: center;
  display: flex;
  font-family: Arial;
  font-size: 4pt;
  font-weight: bold;
  background: white;
}
.main {
  margin-left: 10px;
  padding: 0px 10px;
}
</style>
    </head>
    <body>
        <div class="main">
  <div class="container">
  <div class="grid" id="grid">
      <div class="cell" style="" id="0000x0000"></div>
      <div class="cell" style="" id="0001x0000">1</div>
      <div class="cell" style="" id="0002x0000">2</div>
      <div class="cell" style="" id="0003x0000">3</div>
      <div class="cell" style="" id="0004x0000">4</div>
      <div class="cell" style="" id="0005x0000">5</div>
      <div class="cell" style="" id="0006x0000">6</div>
      <div class="cell" style="" id="0007x0000">7</div>
      <div class="cell" style="" id="0008x0000">8</div>
      <div class="cell" style="" id="0009x0000">9</div>
      <div class="cell" style="" id="0010x0000">10</div>
      <div class="cell" style="" id="0011x0000">11</div>
      <div class="cell" style="" id="0012x0000">12</div>
      <div class="cell" style="" id="0013x0000">13</div>
      <div class="cell" style="" id="0014x0000">14</div>
      <div class="cell" style="" id="0015x0000">15</div>
      <div class="cell" style="" id="0000x0001">1</div>
      <div class="cell" style="" id="0001x0001"><div style="color: rgb(199, 43, 59)">-----</div></div>
      <div class="cell" style="" id="0002x0001"><div style="color: #333333">-----</div></div>
      <div class="cell" style="" id="0003x0001"></div>
      <div class="cell" style="" id="0004x0001"><div style="color: #333333">-----</div></div>
      <div class="cell" style="" id="0005x0001"></div>
      <div class="cell" style="" id="0006x0001"></div>
      <div class="cell" style="" id="0007x0001"></div>
      <div class="cell" style="" id="0008x0001"></div>
      <div class="cell" style="" id="0009x0001"></div>
      <div class="cell" style="" id="0010x0001"></div>
      <div class="cell" style="" id="0011x0001"></div>
      <div class="cell" style="" id="0012x0001"></div>
      <div class="cell" style="" id="0013x0001"></div>
      <div class="cell" style="" id="0014x0001"></div>
      <div class="cell" style="" id="0015x0001"></div>
      <div class="cell" style="" id="0000x0002">2</div>
      <div class="cell" style="" id="0001x0002"></div>
      <div class="cell" style="" id="0002x0002"><div style="color: #333333">-----</div></div>
      <div class="cell" style="" id="0003x0002"><div style="color: rgb(199, 43, 59)">-----</div></div>
      <div class="cell" style="background-color:  orange" id="0004x0002"><div style="color: #333333">-----</div></div>
      <div class="cell" style="background-color:  pink" id="0005x0002"></div>
      <div class="cell" style="" id="0006x0002"></div>
      <div class="cell" style="background-color:  pink" id="0007x0002"></div>
      <div class="cell" style="" id="0008x0002"></div>
      <div class="cell" style="" id="0009x0002"></div>
      <div class="cell" style="" id="0010x0002"></div>
      <div class="cell" style="" id="0011x0002"></div>
      <div class="cell" style="" id="0012x0002"></div>
      <div class="cell" style="" id="0013x0002"></div>
      <div class="cell" style="" id="0014x0002"></div>
      <div class="cell" style="" id="0015x0002"></div>
      <div class="cell" style="" id="0000x0003">3</div>
      <div class="cell" style="" id="0001x0003"></div>
      <div class="cell" style="" id="0002x0003"></div>
      <div class="cell" style="" id="0003x0003"><div style="color: rgb(199, 43, 59)">-----</div></div>
      <div class="cell" style="" id="0004x0003"></div>
      <div class="cell" style="background-color:  pink" id="0005x0003"></div>
      <div class="cell" style="background-color:  orange" id="0006x0003"></div>
      <div class="cell" style="background-color:  pink" id="0007x0003"></div>
      <div class="cell" style="" id="0008x0003"></div>
      <div class="cell" style="" id="0009x0003"></div>
      <div class="cell" style="" id="0010x0003"></div>
      <div class="cell" style="" id="0011x0003"></div>
      <div class="cell" style="" id="0012x0003"></div>
      <div class="cell" style="" id="0013x0003"></div>
      <div class="cell" style="" id="0014x0003"></div>
      <div class="cell" style="" id="0015x0003"></div>
      <div class="cell" style="" id="0000x0004">4</div>
      <div class="cell" style="" id="0001x0004"></div>
      <div class="cell" style="" id="0002x0004"></div>
      <div class="cell" style="" id="0003x0004"></div>
      <div class="cell" style="" id="0004x0004"></div>
      <div class="cell" style="" id="0005x0004"></div>
      <div class="cell" style="background-color:  orange" id="0006x0004"></div>
      <div class="cell" style="" id="0007x0004"></div>
      <div class="cell" style="" id="0008x0004"></div>
      <div class="cell" style="" id="0009x0004"></div>
      <div class="cell" style="" id="0010x0004"></div>
      <div class="cell" style="" id="0011x0004"></div>
      <div class="cell" style="" id="0012x0004"></div>
      <div class="cell" style="" id="0013x0004"></div>
      <div class="cell" style="" id="0014x0004"></div>
      <div class="cell" style="" id="0015x0004"></div>
      <div class="cell" style="" id="0000x0005">5</div>
      <div class="cell" style="" id="0001x0005"></div>
      <div class="cell" style="" id="0002x0005"></div>
      <div class="cell" style="" id="0003x0005"></div>
      <div class="cell" style="" id="0004x0005"></div>
      <div class="cell" style="" id="0005x0005"></div>
      <div class="cell" style="" id="0006x0005"></div>
      <div class="cell" style="" id="0007x0005"></div>
      <div class="cell" style="" id="0008x0005"></div>
      <div class="cell" style="" id="0009x0005"></div>
      <div class="cell" style="" id="0010x0005"></div>
      <div class="cell" style="" id="0011x0005"></div>
      <div class="cell" style="" id="0012x0005"></div>
      <div class="cell" style="" id="0013x0005"></div>
      <div class="cell" style="" id="0014x0005"></div>
      <div class="cell" style="" id="0015x0005"></div>
      <div class="cell" style="" id="0000x0006">6</div>
      <div class="cell" style="" id="0001x0006"></div>
      <div class="cell" style="" id="0002x0006"></div>
      <div class="cell" style="" id="0003x0006"></div>
      <div class="cell" style="" id="0004x0006"></div>
      <div class="cell" style="" id="0005x0006"></div>
      <div class="cell" style="" id="0006x0006"></div>
      <div class="cell" style="" id="0007x0006"></div>
      <div class="cell" style="" id="0008x0006"></div>
      <div class="cell" style="" id="0009x0006"></div>
      <div class="cell" style="" id="0010x0006"></div>
      <div class="cell" style="" id="0011x0006"></div>
      <div class="cell" style="" id="0012x0006"></div>
      <div class="cell" style="" id="0013x0006"></div>
      <div class="cell" style="" id="0014x0006"></div>
      <div class="cell" style="" id="0015x0006"></div>
      <div class="cell" style="" id="0000x0007">7</div>
      <div class="cell" style="" id="0001x0007"></div>
      <div class="cell" style="" id="0002x0007"></div>
      <div class="cell" style="" id="0003x0007"></div>
      <div class="cell" style="" id="0004x0007"></div>
      <div class="cell" style="" id="0005x0007"></div>
      <div class="cell" style="" id="0006x0007"></div>
      <div class="cell" style="" id="0007x0007"></div>
      <div class="cell" style="" id="0008x0007"></div>
      <div class="cell" style="" id="0009x0007"></div>
      <div class="cell" style="" id="0010x0007"></div>
      <div class="cell" style="" id="0011x0007"></div>
      <div class="cell" style="" id="0012x0007"></div>
      <div class="cell" style="" id="0013x0007"></div>
      <div class="cell" style="" id="0014x0007"></div>
      <div class="cell" style="" id="0015x0007"></div>
      <div class="cell" style="" id="0000x0008">8</div>
      <div class="cell" style="" id="0001x0008"></div>
      <div class="cell" style="" id="0002x0008"></div>
      <div class="cell" style="" id="0003x0008"></div>
      <div class="cell" style="" id="0004x0008"></div>
      <div class="cell" style="" id="0005x0008"></div>
      <div class="cell" style="" id="0006x0008"></div>
      <div class="cell" style="" id="0007x0008"></div>
      <div class="cell" style="" id="0008x0008"></div>
      <div class="cell" style="" id="0009x0008"></div>
      <div class="cell" style="" id="0010x0008"></div>
      <div class="cell" style="" id="0011x0008"></div>
      <div class="cell" style="" id="0012x0008"></div>
      <div class="cell" style="" id="0013x0008"></div>
      <div class="cell" style="" id="0014x0008"></div>
      <div class="cell" style="" id="0015x0008"></div>
      <div class="cell" style="" id="0000x0009">9</div>
      <div class="cell" style="" id="0001x0009"></div>
      <div class="cell" style="" id="0002x0009"></div>
      <div class="cell" style="" id="0003x0009"></div>
      <div class="cell" style="" id="0004x0009"></div>
      <div class="cell" style="" id="0005x0009"></div>
      <div class="cell" style="" id="0006x0009"></div>
      <div class="cell" style="" id="0007x0009"></div>
      <div class="cell" style="" id="0008x0009"></div>
      <div class="cell" style="" id="0009x0009"></div>
      <div class="cell" style="" id="0010x0009"></div>
      <div class="cell" style="" id="0011x0009"></div>
      <div class="cell" style="" id="0012x0009"></div>
      <div class="cell" style="" id="0013x0009"></div>
      <div class="cell" style="" id="0014x0009"></div>
      <div class="cell" style="" id="0015x0009"></div>
      <div class="cell" style="" id="0000x0010">10</div>
      <div class="cell" style="" id="0001x0010"></div>
      <div class="cell" style="" id="0002x0010"></div>
      <div class="cell" style="" id="0003x0010"></div>
      <div class="cell" style="" id="0004x0010"></div>
      <div class="cell" style="" id="0005x0010"></div>
      <div class="cell" style="" id="0006x0010"></div>
      <div class="cell" style="" id="0007x0010"></div>
      <div class="cell" style="" id="0008x0010"></div>
      <div class="cell" style="" id="0009x0010"></div>
      <div class="cell" style="" id="0010x0010"></div>
      <div class="cell" style="" id="0011x0010"></div>
      <div class="cell" style="" id="0012x0010"></div>
      <div class="cell" style="" id="0013x0010"></div>
      <div class="cell" style="" id="0014x0010"></div>
      <div class="cell" style="" id="0015x0010"></div>
      <div class="cell" style="" id="0000x0011">11</div>
      <div class="cell" style="" id="0001x0011"></div>
      <div class="cell" style="" id="0002x0011"></div>
      <div class="cell" style="" id="0003x0011"></div>
      <div class="cell" style="" id="0004x0011"></div>
      <div class="cell" style="" id="0005x0011"></div>
      <div class="cell" style="" id="0006x0011"></div>
      <div class="cell" style="" id="0007x0011"></div>
      <div class="cell" style="" id="0008x0011"></div>
      <div class="cell" style="" id="0009x0011"></div>
      <div class="cell" style="" id="0010x0011"></div>
      <div class="cell" style="" id="0011x0011"><div style="color: rgb(199, 43, 59)">|</div></div>
      <div class="cell" style="" id="0012x0011"><div style="color: rgb(199, 43, 59)">|</div></div>
      <div class="cell" style="" id="0013x0011"><div style="color: rgb(199, 43, 59)">|</div></div>
      <div class="cell" style="" id="0014x0011"><div style="color: rgb(199, 43, 59)">|</div></div>
      <div class="cell" style="" id="0015x0011"><div style="color: rgb(199, 43, 59)">|</div></div>
      <div class="cell" style="" id="0000x0012">12</div>
      <div class="cell" style="" id="0001x0012"></div>
      <div class="cell" style="" id="0002x0012"></div>
      <div class="cell" style="" id="0003x0012"></div>
      <div class="cell" style="" id="0004x0012"></div>
      <div class="cell" style="" id="0005x0012"></div>
      <div class="cell" style="" id="0006x0012"></div>
      <div class="cell" style="" id="0007x0012"></div>
      <div class="cell" style="" id="0008x0012"></div>
      <div class="cell" style="" id="0009x0012"></div>
      <div class="cell" style="" id="0010x0012"></div>
      <div class="cell" style="" id="0011x0012"></div>
      <div class="cell" style="" id="0012x0012"></div>
      <div class="cell" style="" id="0013x0012"></div>
      <div class="cell" style="" id="0014x0012"></div>
      <div class="cell" style="" id="0015x0012"></div>
      <div class="cell" style="" id="0000x0013">13</div>
      <div class="cell" style="" id="0001x0013"></div>
      <div class="cell" style="" id="0002x0013"></div>
      <div class="cell" style="" id="0003x0013"></div>
      <div class="cell" style="" id="0004x0013"></div>
      <div class="cell" style="" id="0005x0013"></div>
      <div class="cell" style="" id="0006x0013"></div>
      <div class="cell" style="" id="0007x0013"></div>
      <div class="cell" style="" id="0008x0013"></div>
      <div class="cell" style="" id="0009x0013"></div>
      <div class="cell" style="" id="0010x0013"></div>
      <div class="cell" style="" id="0011x0013"><div style="color: rgb(199, 43, 59)">|</div></div>
      <div class="cell" style="" id="0012x0013"><div style="color: #333333">|</div></div>
      <div class="cell" style="" id="0013x0013"><div style="color: rgb(199, 43, 59)">|</div></div>
      <div class="cell" style="" id="0014x0013"><div style="color: #333333">|</div></div>
      <div class="cell" style="" id="0015x0013"><div style="color: rgb(199, 43, 59)">|</div></div>
      <div class="cell" style="" id="0000x0014">14</div>
      <div class="cell" style="" id="0001x0014"></div>
      <div class="cell" style="" id="0002x0014"></div>
      <div class="cell" style="" id="0003x0014"></div>
      <div class="cell" style="" id="0004x0014"></div>
      <div class="cell" style="" id="0005x0014"></div>
      <div class="cell" style="" id="0006x0014"></div>
      <div class="cell" style="" id="0007x0014"></div>
      <div class="cell" style="" id="0008x0014"></div>
      <div class="cell" style="" id="0009x0014"></div>
      <div class="cell" style="" id="0010x0014"></div>
      <div class="cell" style="" id="0011x0014"></div>
      <div class="cell" style="" id="0012x0014"></div>
      <div class="cell" style="" id="0013x0014"></div>
      <div class="cell" style="" id="0014x0014"></div>
      <div class="cell" style="" id="0015x0014"></div>
      <div class="cell" style="" id="0000x0015">15</div>
      <div class="cell" style="" id="0001x0015"></div>
      <div class="cell" style="" id="0002x0015"></div>
      <div class="cell" style="" id="0003x0015"></div>
      <div class="cell" style="" id="0004x0015"></div>
      <div class="cell" style="" id="0005x0015"></div>
      <div class="cell" style="" id="0006x0015"></div>
      <div class="cell" style="" id="0007x0015"></div>
      <div class="cell" style="" id="0008x0015"></div>
      <div class="cell" style="" id="0009x0015"></div>
      <div class="cell" style="" id="0010x0015"></div>
      <div class="cell" style="" id="0011x0015"></div>
      <div class="cell" style="" id="0012x0015"></div>
      <div class="cell" style="" id="0013x0015"></div>
      <div class="cell" style="" id="0014x0015"></div>
      <div class="cell" style="" id="0015x0015"></div>
  </div>
</div>
        </div>
    </body>
</html>
